web-portal/internal-api/upload-managers/S3Uploader.js

```diff
@@ -1,16 +1,11 @@
-const aws = require('aws-sdk')
+const { S3 } = require("@aws-sdk/client-s3")
```

Comment:
Tiny thing, but the rest of the web portal code is/should be using single quotes (linter setting).